.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
          pyver: ['3.9', '3.10', '3.11', '3.12', '3.13']
          experimental: [false]
          include:
-           - pyver: "3.14"
```

Comment:
This would use the latest release already, so this PR has just stuck us with an older release...